### DISCUSSION TITLE
🐛 Bug: border-gray-200 not working

### DISCUSSION THREAD

**Discussion by nermalcat69** [tailwindlabs/tailwindcss#12452] [Help]
**What version of Tailwind CSS are you using?**
v3.3.5

**What build tool (or framework if it abstracts the build tool) are you using?**
just nextjs and postcss (8.4.31)

**What version of Node.js are you using?**
v20.8.1

**What browser are you using?**

For example: Chrome, Safari, or N/A

**What operating system are you using?**
macOS

**Reproduction URL**

[border-gray-200](https://nermalcat69-bo80xyird-ciaoarjun.vercel.app)

[border-gray-300](https://nermalcat69-233iroff5-ciaoarjun.vercel.app)

**Describe your issue**

border-gray-200 isnt working, its a lot darker than as compared to border-gray-300 or 400

border-gray-200

<img width="506" alt="Screenshot 2023-11-20 at 10 57 55 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/73933669/b0771f55-ba5f-4ffe-8ab4-cb55e6d35d75">

border-gray-300
<img width="523" alt="Screenshot 2023-11-20 at 10 59 09 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/73933669/bf8e8b9a-4452-49ef-a372-b066f321ae87">

**Comment by alex-krasikau** [comment]
Hey @nermalcat69!

There's a guide to help you investigate the issue, can you please go throw the steps listed in the article and take an extra attention to the 3rd point - https://dev.to/plainsailing/tailwind-css-not-working-its-probably-this-9m4?

If that doesn't help, get back and we will try to figure out what's wrong.
